src/components/BluetoothPatternInput.tsx

```diff
@@ -165,7 +165,6 @@ const PatternColumnInput = ({
           <FormattedMessage
             id="connect-pattern-input-label"
             values={{ colNum: colIdx + 1 }}
-            sr-onlu
```

Comment:
It's already in a VisuallyHidden component, and this prop obviously does nothing.